src/styles/blocks/stars.css

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(/src/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). Since `.stars__star` elements are likely to be displayed in a row, it's good to have a right margin, but ensure that you do not add both vertical and horizontal margins to avoid potential margin collapse issues.